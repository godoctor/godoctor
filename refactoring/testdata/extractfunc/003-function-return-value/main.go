// <<<<< extract,9,2,11,38,changeB,pass
package main

import "fmt"

func main() {
	a := 5
	b := 4
	fmt.Println("Value of a and b", a, b)
	b = 77
	fmt.Println("Modified Value of b", b)
}
