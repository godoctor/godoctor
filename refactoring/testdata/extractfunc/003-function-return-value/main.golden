// <<<<< extract,9,2,11,38,changeB,pass
package main

import "fmt"

func main() {
	a := 5
	b := 4
	changeB(a, b)
}
func changeB(a int, b int) {
	fmt.Println("Value of a and b", a, b)
	b = 77
	fmt.Println("Modified Value of b", b)
}
